client/components/mma/accountoverview/AccountOverview.tsx

```diff
@@ -197,6 +198,16 @@ const AccountOverviewPage = ({ isFromApp }: IsFromAppProps) => {
 		return specificProductType.groupedProductType;
 	};
 
+	const possiblyAffectedByCanadaPostStrike = allActiveProductDetails.some(
+		(product) => {
+			return (
+				(product.tier === 'Tier Three' ||
+					product.tier === 'Guardian Weekly - ROW') &&
+				product.billingCountry === 'Canada'
```

Comment:
as discussed we could use product.deliveryAddress.country here (if it's getting through) to make it more accurate.  That (from MDAPI) contains the mailing address from salesforce.
It might also be worth toLowerCaseing it as there are a notable amount of `CANADA`s (not to mention a few `CA`s

Reply:
👍  ... yep I've changed to use the delivery address instead of the billing address and accounted for differing cases and 'CA'